gato/github/api.py

```diff
@@ -667,7 +703,8 @@ def retrieve_run_logs(self, repo_name: str, short_circuit: str = True):
         start_date = datetime.now() - timedelta(days = 60)
         runs = self.call_get(
             f'/repos/{repo_name}/actions/runs', params={
-                "per_page": "30",
+                # "per_page": "30",
```

Comment:
Set back to 30 unless there is a reason for 10. 30 is a good balance between speed and not having a false negative (planning to have this line up with enum profiles (#54), but that's a later task.